app/components/DownloadApps/index.js

```diff
@@ -0,0 +1,101 @@
+import React, { Component } from 'react'
```

Comment:
The same comment I wrote to "Welcome" page applies here as well.
General folders like src/components or utils should be kept them as smaller and simpler as possible. The main idea is scalability and maintainability of the project.